### PR TITLE
fix(grammar): correct plural/possessive usage of "'s"

### PR DESCRIPTION
**Pull request by jsejcksn**
The method I used to make the changes in this PR:

I searched for string matches of `"'s"` in all files and reviewed each result, replacing instances with `"s"` or `"es"` as appropriate. I skipped the results which were matches for the start of string values or were correct usages (e.g. possessive/contractions, etc.). However, please review for accuracy.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1732,7 +1732,7 @@ bar.txt
 
 - **command:** export default types
   ([34fcddd](https://github.com/c4spar/deno-cliffy/commit/34fcddd))
-- **command:** make `complete` method in custom type class's optional.
+- **command:** make `complete` method in custom type classs optional.
```
Comment (robinmorton) [inline CHANGELOG.md:1735]: I think this one should be `classes`

**Comment by c4spar** [comment]
thx @jsejcksn and thx for the review @robinmorton 👍. 
@jsejcksn could you update your PR please? I messed up the ci workflow last time so they don't run on pull requests, but that's fixed in the main branch now.

**Comment by jsejcksn** [comment]
Thanks for the review @robinmorton. I've made another commit including your suggestions.

**Review by c4spar — APPROVED** [review]
Thx @jsejcksn 👍, LGTM!

**Pull request merged**